### DISCUSSION TITLE
[Feature]: Notification on new login

### DISCUSSION THREAD

**Discussion by JVT038** [immich-app/immich#1632] [Feature Request]
### Feature detail

When logging in with my account from a new device / client, I would like to be notified by this.
My videos and photos are very important to me, and I want to be warned if a random person somehow got access to my Immich account. To alert a user, an e-mail should be sent to the user's e-mail address. The email should say something like:

> A new login has been detected in your account. The IP was 12.34.567.89 and the device was [Insert Chrome / Firefox / Android / iOS / any client]. If this was you, you don't have to do anything, blablabla. If this wasn't you, reset your password.

I want to be notified when someone signs in through any client, so for web and mobile.

### Platform

Web

**Comment by bo0tzz** [comment]
Duplicate of #1615
